middleware/auth.ts

```diff
@@ -6,4 +6,9 @@ export default defineNuxtMiddleware((context) => {
    * @todo Redirect the user to main page if token is present in store
    * @hints check what propeties context has
    */
+  const {redirect,store}=context;
+  if(!store.getters.auth){
+    return redirect('/login/');
+  }
```

Comment:
if the user is logged in he should also be prevented from going to login. Destructuring looks good